src/frontend/efiling-frontend/src/domain/package/package-review/PackageReview.stories.js

```diff
@@ -4,6 +4,9 @@ import PackageReview from "./PackageReview";
 jest.mock("react-router-dom", () => ({
   ...jest.requireActual("react-router-dom"),
   useParams: jest.fn().mockReturnValue({ packageId: 1 }),
+  useLocation: jest
+    .fn()
+    .mockReturnValue({ search: "?returnUrl=http://www.google.com" }),
```

Comment:
Could we add a test with an encoded url that has query parameters, make sure we can handle it?